walkthroughs/cdk-examples/howto-mutual-tls-file-provided-by-acm/test/howto-mutual-tls-file-provided-by-acm.test.ts

```diff
@@ -0,0 +1,17 @@
+// import * as cdk from 'aws-cdk-lib';
```

Comment:
It is expected to have this test file as a placeholder?

Reply:
Nice catch! we dont need this, i will delete this file. Keeping the directory in case we add test cases later